docs/docsite/rst/community/steering/steering_committee_membership.rst

```diff
@@ -130,14 +130,14 @@ In the case of starting the removal process, the topic's description in the reas
 Chairperson
 ------------
 
-The chairperson election will happen once a year around the time of
-Ansible Fest. If the current chairperson has to step down early, the election happens immediately.
+The chairperson election happens once a year at the time the Committee agrees on.
```

Comment:
This part is unclear to me. The committee will agree on a specific time of year that we will use every year? (in which case, that should be decided first and then put in this PR)

Or just whenever we feel like it we can have a new election? (in which case, it probably means we need to have a vote to have an election)

Reply:
@briantist i wouldn't specify a fixed time here because we can easily miss it, so added voting. PTAL